### PR TITLE
Set asm.tabs to 6

### PR DESCRIPTION
**Pull request by ITAYC0HEN**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [rizin book](https://github.com/rizinorg/rizinbook) with the relevant information (if needed)

**Detailed description**
This PR increase `asm.tabs` from 0 to 6

**Before:**
image

**After:**

image

This value sets the number of spaces from the first letter of the instruction, to the first operand. So instructions with more then 6 chars will break the alignment. Before this PR,  instructions with more than 3 chars breaks the alignment

### CODE REVIEW

**Comment by ret2libc** [comment]
~~I don't like the extra (and unnecessary) space after the first operand. Otherwise it is fine for me.~~

**Comment by ITAYC0HEN** [comment]
The screenshot I put is wrong.will update tomorrow.
The pr enables asm.tabs.once that doesn't add spaces after operands

**Review by ret2libc — APPROVED** [review]
It looks nicer to me, but I suggest this gets all the 6 approvals before merging.

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]
It might become problematic on super-long instructions probably, like some AVX instructions or TMS320CXX ones.

**Comment by ITAYC0HEN** [comment]
On AVX it still looks better than the default `asm.tabs=0`. I tested

**Review by thestr4ng3r — APPROVED** [review]

**Review by kazarmy — APPROVED** [review]

**Review by yossizap — APPROVED** [review]
Much better :+1:

**Comment by kazarmy** [comment]
Tests need to be fixed, but something like `pdr~addu t0[7]` is going to be very sensitive to disasm format imo.

File: test/db/cmd/cmd_pdr [hunk 1]
```diff
@@ -4,11 +4,11 @@ CMDS=<<EOF
 e asm.arch=x86
 e asm.bits=64
 e asm.bytes=true
-e asm.fcnlines=false
+e asm.lines.fcn=false
 wx 48c7c01000000075084805b8000000eb064805e60000004889c7c3
```
Comment (kazarmy) [inline test/db/cmd/cmd_pdr:8]:
```suggestion
e asm.tabs=0
wx 48c7c01000000075084805b8000000eb064805e60000004889c7c3
```

File: test/db/cmd/cmd_pdr [hunk 2]
```diff
@@ -4,11 +4,11 @@ CMDS=<<EOF
...
 af
 e asm.emu=true
-pdr~add rax[6]
+pdr~add   rax[6]
```
Comment (kazarmy) [inline test/db/cmd/cmd_pdr]:
```suggestion
pdr~add rax[6]
```

File: test/db/cmd/cmd_pdr [hunk 3]
```diff
@@ -4,11 +4,11 @@ CMDS=<<EOF
 e asm.arch=x86
 e asm.bits=64
 e asm.bytes=true
-e asm.fcnlines=false
+e asm.lines.fcn=false
```
Comment (kazarmy) [inline test/db/cmd/cmd_pdr:7]:
```suggestion
e asm.lines.fcn=false
e asm.tabs=0
```

File: test/db/cmd/cmd_pdr [hunk 4]
```diff
@@ -24,10 +24,10 @@ e asm.arch=mips
 e analysis.gpfixed=false
 wx 10000824cc009c270500281500000000200c082521401c010a00000800000000300c082521401c010800e003
 af
-e asm.fcnlines=false
+e asm.lines.fcn=false
```
Comment (kazarmy) [inline test/db/cmd/cmd_pdr:28]:
```suggestion
e asm.lines.fcn=false
e asm.tabs=0
```

File: test/db/cmd/cmd_pdr [hunk 5]
```diff
@@ -24,10 +24,10 @@ e asm.arch=mips
...
 e asm.emu=true
 e analysis.gp=0xabc
-pdr~addu t0[7]
+pdr~addu  t0[7]
```
Comment (kazarmy) [inline test/db/cmd/cmd_pdr]:
```suggestion
pdr~addu t0[7]
```

**Review by kazarmy — CHANGES_REQUESTED** [review]
Doesn't seem very likely that someone will be counting spaces tbh.

**Review by kazarmy — APPROVED** [review]

**Pull request merged**